content/design/line-styles/index.html

```diff
@@ -2,65 +2,136 @@
-    code: '@include bb-border(light)'
-    example: '<div class="bb-subsection-heading" style="border-bottom: 1px solid #e7eaec;">Light separator</div>'
-    styling:
-      - '1px solid #e7eaec'
-      - When this line style subdivides content within a container, use a 15px margin on both ends.
-
-  - usage:
-    - '<p>The dark separator is a moderate gray, single-pixel line. This heavier stroke holds up between two different colors, which makes it ideal for borders and boundaries. It also frequently separates active content from passive content. For example, it can separate a toolbar from the content that the toolbar acts on.</p><p>SKY UX uses this style for:</p><ul><li>Borders on <a href="../../components/buttons">buttons</a></li><li>Top and bottom boundaries for toolbars in areas such as <a href="../../components/tiles">tiles</a> and <a href="../../components/grids">grids</a></li><li>Column dividers for header rows in <a href="../../components/grids">grids</a></li><li>Borders for <a href="../../components/card">cards</a></li><li>Built-in data visualization axes</li></ul>'
-    code: '@include bb-border(dark)'
-    example: '<div class="bb-subsection-heading" style="border-bottom: 1px solid #ccc;">Dark separator</div>'
-    styling: '1px solid #ccc'
-
-itemsIsolated:
-  - usage: This line style highlights the active navigation item in the <a href="../../components/navbar">navbar</a>. It aligns to the bottom of the navigation item and stretches to its full width.
-    code: '@include bb-border(nav)'
-    example: '<div class="bb-subsection-heading" style="border-bottom: 4px solid #1c84c6;">Navigation selection underline</div>'
-    styling: '4px solid #1c84c6'
+<h1 class="bb-page-heading">Line styles</h1>
 
-  - usage: This line style separates <a href="../../components/tabset">tab</a> controls from their contents. It aligns to the bottom of the tab controls and stretches to the full width of the container.
-    code: '@include bb-border(tab)'
-    example: '<div class="bb-subsection-heading" style="border-bottom: 2px solid #1c84c6;">Tab underline</div>'
-    styling: '2px solid #1c84c6'
+<p>SKY UX uses various line styles to bound, divide, and highlight content. The lines are designed to maintain visual hierarchy regardless of how complex pages become or how many nested containers they include.</p>
+<p>As with other types of formatting or decoration in SKY UX, use lines minimally. Remember that white space and positioning can group or highlight content without extra subdivisions or boundaries.</p>
+<p>SKY UX uses 3 generalized line styles that you can apply with a SASS mixin.</p>
```

Comment:
Why did we remove the link to https://github.com/blackbaud/skyux/blob/master/scss/_mixins.scss from the "a SASS mixin" text? Just making sure that was intentional.